models/camelot/camelot_pair_fee_rates_schema.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+
+models:
+  - name: camelot_pair_fee_rates
```

Comment:
be sure to update spell name here, if file change above

Reply:
also plz add unique combo of columns test in here, it should match the unique keys in model exactly